expression/column.go

```diff
@@ -490,6 +490,15 @@ func (col *Column) HashCode(_ *stmtctx.StatementContext) []byte {
 	return col.hashcode
 }
 
+// CleanHashCode will clean the hashcode you may be cached before. It's used especially in schema-cloned & reallocated-uniqueID's cases.
+func (col *Column) CleanHashCode() {
+	if len(col.hashcode) != 0 {
+		col.hashcode = col.hashcode[:0]
+	} else {
+		col.hashcode = make([]byte, 0, 9)
```

Comment:
I think we can just use this code branch.

the `[:0]` won't clean the memory usage. The two different unique id might refer to the same memory.

Reply:
make sense